docs/en/users/06_Fever_API.md

```diff
@@ -6,7 +6,7 @@ and general aspects of API access.
 ## RSS clients
 
 There are many RSS clients existing supporting Fever APIs but they seem to understand the Fever API a bit differently.
-If your favourite client does not work properly with this API, create an issue and we will have a look.
+If your favorite client does not work properly with this API, create an issue and we will have a look.
```

Comment:
That was not a typo, just UK spelling 🤓

Reply:
I know, I just made it more international :) You can revert this if you want to keep it consistent with other documentation.